enterprise_extensions/hypermodel.py

```diff
@@ -1,26 +1,21 @@
 # -*- coding: utf-8 -*-
-from __future__ import (absolute_import, division,
-                        print_function)
-import numpy as np
-import os
-import scipy.stats as scistats
-import scipy.linalg as sl
+
 import os
 
-from enterprise import constants as const
-from enterprise.signals import signal_base
+import numpy as np
+import scipy.linalg as sl
 
 try:
-    import cPickle as pickle
```

Comment:
It seems pickle is not used at all so the `try`/`except` block can be removed entirely.

Reply:
Yeah, this is a weird output from `autoflake`. It knows the imports aren't needed, but isn't sure what to do about the `try/except`. I removed them in the old PR by hand, and just missed them this time.